types/alumni-companies.ts

```diff
@@ -0,0 +1,16 @@
+import { SanityImageObject } from '@sanity/image-url/lib/types/types';
+
+export type ArrayAlumniCompanies = {
+  _key: string;
+  name: string;
+  picture: SanityImageObject;
+};
+
+export type AlumniCompaniesFetchT = {
+  _id: string;
+  companies: Array<ArrayAlumniCompanies>;
```

Comment:
Array<ArrayAlumn , let's avoid duplication :)

Reply:
Also, companies should contain Array<AlumniCompany>, not companies (because it's already array of companies)

Reply:
Array<AlumniCompany>;